apps/fabric-website/index-devx.html

```diff
@@ -0,0 +1,98 @@
+<style>
+  body,
+  html {
+    margin: 0;
+    padding: 0;
+  }
+
+  .loading {
+    align-items: center;
+    background-color: #212121;
+    border-top: 50px solid #000;
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+  }
+</style>
+<div id="main">
+  <div class="loading">
+    <img class="loadingImage" src="https://static2.sharepointonline.com/files/fabric/fabric-website/images/loading.gif" alt="Loading"
+      width="32" height="32" />
+  </div>
+</div>
+
+<script type="text/javascript">
+  var isProduction = false;
+  var isDogfood = false;
+  var isLocal = false;
+  var today = new Date();
+  var date = today.getFullYear() + '-' + (today.getMonth() + 1) + '-' + today.getDate();
+  var entryPointFilename = 'fabric-sitev5';
```

Comment:
nit: Feels like this should be treated as a const, e.g. `var ENTRY_POINT_FILENAME` or something like that.